include/reactor-cpp/trace.hh

```diff
@@ -99,15 +64,15 @@ TRACEPOINT_EVENT(
 #else
 
 #ifndef _REACTOR_CPP_TRACE_H
-#define _REACTOR_CPP_TRACE_H
```

Comment:
This shouldn't change (or `_REACTOR_CPP_TRACE_H` needs to be renamed everywhere)